dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.internal.grpc-1.0.feature

```diff
@@ -2,16 +2,18 @@
 symbolicName=io.openliberty.internal.grpc-1.0
 singleton=true
 IBM-API-Package: \
-  com.google.common.util.concurrent; type="stable",\
-  com.google.protobuf;  type="stable",\
+  com.google.common.collect; type="internal",\
+  com.google.common.util.concurrent; type="internal",\
+  com.google.protobuf;  type="internal",\
   io.grpc;  type="stable", \
```

Comment:
I thought that usually the IBM-API-Package was listed on the public API feature (so that the packages are only exposed to users if the public feature is enabled, and not if the private feature is pulled in by another feature).

I'm assuming in this case, this feature is only used by `grpc` and `grpcClient`?

Reply:
You're right, this private feature is only used by the public features `grpc-1.0` and `grpcClient-1.0`.  My intention here was to provide the common APIs for those features from a single internal feature.